tests/integration/targets/init.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2155,SC2086
 
-BASE_DIR=$(dirname "${BASH_SOURCE[0]}")
-export ANSIBLE_ROLES_PATH=${BASE_DIR}
+export VALIDATED_COLLECTIONS_PATH="/home/kni/actions-runner/eco-vcenter/cloud.vmware_ops/cloud.vmware_ops/ansible_collections"
```

Comment:
There is an environment variable set when running on github actions, GITHUB_WORKSPACE. I think its better to use that instead of hard coding the path.
Im not sure if its
"${GITHUB_WORKSPACE}/cloud.vmware_ops/ansible_collections"
or
"${GITHUB_WORKSPACE}/ansible_collections"

Reply:
hmm, seems like `GITHUB_WORKSPACE` is set to a value which is not the excepted one (something with /home/runner...), also, this value is removed after moving from the action yml to the` init.sh `

Reply:
OK I found that `ANSIBLE_COLLECTIONS_PATH` does the trick here